maas/client/viscera/machines.py

```diff
@@ -193,6 +193,12 @@ class Machine(Object, metaclass=MachineType):
         data = await self._handler.release(**params)
         return type(self)(data)
 
+    async def get_power_parameters(self):
+        data = await self._origin.Machines._handler.power_parameters(
+            id=[self.system_id]
+        )
+        return data
+
```

Comment:
There is a built-in power-parameters operation for a machine. I would think that:

```
data = await self._handler.power_parameters()
return data
```

At the moment your code would return a list. At least you would need to do `return data[0]`